src/Raven.Client/Constants.cs

```diff
@@ -103,6 +103,15 @@ private Configuration()
             public const string ClientId = "Configuration/Client";
         }
 
+        public class Counters
+        {
+            private Counters()
+            {
+            }
+
+            public const string All = "@allCounters";
```

Comment:
Let's match the `@all_docs` and change this to `@all_counters`

Reply:
done